app/views/layouts/mailers/_service_plus.html.haml

```diff
@@ -0,0 +1,10 @@
+%div{ style: 'background-color: #F5F5FE; padding: 15px;' }
+  %table{ width: "100%", border: "0", cellspacing: "0", cellpadding: "0" }
+    %tr
+      %td{ width: "25%" }
+        = image_tag('mailer/logo-services-plus.svg', "aria-hidden": true, style: 'vertical-align: middle')
```

Comment:
Faudrait tester un envoi réel de mail par exemple depuis la preprod car j'ai plusieurs doutes : 

- le support manquant du svg de certains clients mails (quasi sûr qu'il faut utiliser une image en png voir jpg)
- `image_tag` dans un email, je ne sais pas si génère un chemin absolu avec le domaine ? Si c'est pas le cas faudrait passer l'url de l'image, peut-être comme ça : `image_tag(image_url('mailer/logo…'), …)` C'est ce qu'on fait pour les logos (d'ailleurs je vois qu'il y a aussi du style en + pour augmenter la compatibilité des clients mails mais là j'ai aucune idée ce qu'il faudrait mettre…)

Reply:
oui... pb image... accessibilité etc...

Reply:
Ok j'ai hésité pour le svg mais je me suis fait avoir par l'image `confirmation-email.svg` - qui a l'air d'être utilisé, mais peut-être dans un autre contexte du coup.

Reply:
d'après cet article moins de 60% de support du svg en 2022 (notamment pas gmail). https://css-tricks.com/a-guide-on-svg-support-in-email/
Je ferai un PNG et fixerai celui du confirmation